services/provider/proto/provider.proto

```diff
@@ -26,10 +26,10 @@ service OCSProvider {
   rpc AcknowledgeOnboarding(AcknowledgeOnboardingRequest)
   returns (AcknowledgeOnboardingResponse){}
 
-  // FulfillStorageClassClaim RPC call to create the StorageclassClaim CR on
+  // StorageClassRequest RPC call to create the StorageclassClaim CR on
```

Comment:
No change is required in proto file as well

Reply:
Is this supposed to change terminology? We are still fulfilling a storageclassclaim (not request) so I am not sure why is this change here.